hknweb/candidate/templates/candidate/index.html

```diff
@@ -120,7 +120,27 @@ <h4>Bit-Byte</h4>
         <br>
         <section>
             <h2>Upcoming Events</h2>
-            <p><a href="{% url 'events:index' %}">(see calendar)</a></p>
+            {% if upcoming_events %}
+            <table style="width: 100%;">
+                <thead>
+                    <th>Event</th>
+                    <th>Start Time</th>
+                    <th>End Time</th>
+                    <th>Location</th>
+                </thead>
+
+                <tbody>
+                {% for event in upcoming_events %}
+                    <tr>
+                        <td align="center"> {{ event.name }} </td>
```

Comment:
Did you mean style=“text-align:center” instead of align=“center”?

As a side note, if it’s a table of names, left aligning usually looks better than center aligning.

Reply:
Fixed